.env.test

```diff
@@ -6,6 +6,12 @@ REACT_APP_AUTH_TOKEN=csrfspoof
 REACT_APP_AUTH_HEADER=X-CSRFToken
 REACT_APP_AJAX_TIMEOUT=60000
 
+REACT_APP_CONFIG_SERVICE_LOCALES_DEFAULT_LNG=en
+REACT_APP_CONFIG_SERVICE_LOCALES_DEFAULT_LNG_DESC=English
+REACT_APP_CONFIG_SERVICE_LOCALES=/locales/locales.json
+REACT_APP_CONFIG_SERVICE_LOCALES_PATH=/locales/{{lng}}.json
+REACT_APP_CONFIG_SERVICE_LOCALES_EXPIRE=604800000
+
```

Comment:
Any way to have a `.env.base` or something where all these can get base settings from so that we only need the overrides in these files?

Reply:
looks like we could try using just using a `.env` that would combine `.test` `.production` and `.staging` I'll test it out may make that a follow up task since it's more related to the build, can combine it with #50